app/src/main/java/com/android/sample/model/recipe/Recipe.kt

```diff
@@ -1,12 +1,11 @@
 package com.android.sample.model.recipe
 
 data class Recipe(
```

Comment:
Even though the attributes of the data class are explicit, I think it should be helpful for the others to have a documentation of the data class where you explain what each attribute represents